test/Validator/PhoneNumberTest.php

```diff
@@ -3164,10 +3164,12 @@ public function testAllowPossibleSetterGetter()
         $this->assertTrue($this->validator->allowPossible());
     }
 
-    public function testSetCountryMethodIsCaseInsensitive()
```

Comment:
Your changes make sense. We'll want to be able to retrieve the original version, but also validate in a case-insensitive manner. This is a good solution.